converter.js

```diff
@@ -128,22 +126,43 @@ const output = './out.html';
       setTimeout(resolve, ms)
     })
   }
-
+  
+  var pageContentAfterSerialize = '';
   while (true) {
+    clearTerminal()
     var mathDone = false
     mathDone = await page.evaluate(() => {
       const c = window.__c
+      let msg = document.getElementById('MathJax_Message')
+      if(msg && msg.innerText !== ''){
+        console.log(`Progress: ${document.getElementById('MathJax_Message').innerText}`)
+      }
       return c.done
     })
     await sleep(1000)
     if (mathDone) {
+      clearTerminal()
+      console.log('Serializing document...')
+      pageContentAfterSerialize = await page.evaluate(() => {
+        var s = new XMLSerializer()
+        var d = document
+        var str = s.serializeToString(d)
+        return str;
+      })
       break
     }
   }
 
-  //   console.log("MathJax Status: " + math_status);
-  const htmlOut = await page.content()
-  await writeFile(output, htmlOut)
+  //const htmlOut = await page.content()
+
+  console.log('Saving file...')
+  await writeFile(output, pageContentAfterSerialize)
+  clearTerminal()
+  console.log(`Content saved. Open ${output} to see converted file.`)
```

Comment:
In other projects we use the `LOG_LEVEL=debug` or `trace` or `info` or `error` to set which logging to output. Could you include that in here rather than just `console.log`?

Example:

```
require('dotenv').configure()
x=require('bunyan')
log = new x()

...
log....
```